Classes/Controller/BackendServiceController.php

```diff
@@ -243,6 +244,11 @@ public function publishAction(array $nodeContextPaths, string $targetWorkspaceNa
                     'Node could not be published, probably because of a missing parentNode. Please check that the parentNode has been published.',
                     1682762156
                 );
+            } catch (NodeAggregateDoesCurrentlyNotCoverDimensionSpacePoint $e) {
+                throw new NodeAggregateDoesCurrentlyNotCoverDimensionSpacePoint(
```

Comment:
Thanks for improving the error ;)

I find the pattern of rethrowing the same error class a bit odd. I see that this pattern was used also in the code above and introduced via https://github.com/neos/neos-ui/pull/3468

But I like to suggest to use a generic RuntimeException and additionally pass the original exception as third parameter for both cases.

Reply:
I had a look and all of the exceptions are domain exceptions, so it would make more sense to rethrow that to me. 
Looks like this now: 

![Bildschirmfoto 2023-12-08 um 09 08 32](https://github.com/neos/neos-ui/assets/91674611/bc4cb048-e4d2-410c-818b-08ecb37d85d1)